### PR TITLE
This is all my testing done to the Question package

### PR DESCRIPTION
**Pull request by DennisY-H**

### CODE REVIEW

**Review by AdamDKing — CHANGES_REQUESTED** [review]
Read the README, follow the style guide.

**Comment by DennisY-H** [comment]
We have updated the styling of the test suites.

**Review by DennisY-H — COMMENTED** [review]
I believe I have properly stylized the script.

File: src/test/java/com/revaturelabs/ask/AskApplicationControllerTests.java [hunk 1]
```diff
@@ -299,10 +333,300 @@ public void testCreateQuestion() {
...
+    assertTrue(exampleQuesiton.getHighlightedResponseId() == 1
+        & exampleQuastion2.getHighlightedResponseId() == null);
+    assertTrue(exampleQuesiton.getId() == 1 & exampleQuastion2.getId() == null);
+    assertTrue(exampleQuesiton.getQuestionerId() == 1 & exampleQuastion2.getQuestionerId() == null);
+    assertTrue(exampleQuesiton.getResponses() != null & exampleQuastion2.getResponses() == null);
+    assertTrue(exampleQuesiton.getUser() != null & exampleQuastion2.getUser() == null);
+    assertTrue(exampleQuesiton.getCreationDate() != exampleQuastion2.getCreationDate()
+        & exampleQuastion2.getCreationDate() == null);
+    assertTrue(exampleQuesiton.getImages() != null & exampleQuastion2.getImages() != null);
+    assertTrue(exampleQuesiton.getClass() == exampleQuastion2.getClass());
```
Comment (russchristina) [inline src/test/java/com/revaturelabs/ask/AskApplicationControllerTests.java]: @DennisY-H have you considered using some sort of looping structure to accomplish the initialization? It seems oddly wasteful to do it this way. Not only that, but you could have created your mock data in one central place and just reused it in each test.

**Review by russchristina — DISMISSED** [review]

**Review by russchristina — APPROVED** [review]

**Pull request merged**